explorer/README.md

```diff
@@ -200,33 +200,71 @@ The following options can be passed as a comma-separated list to the
 
 ### State of the Program
 
-The state of the program is printed in the following format, which consists of
-two components: (1) a stack of actions and (2) a memory.
+The state of the program is represented by the memory and the stack. The memory
+is a mapping of addresses to values, and the stack is a list of actions.
 
-    {
-    stack: action1 ## action2 ## ...
-    memory: 0: valueA, 1: valueB, 2: valueC, ...
-    }
+The state of the program is constantly changing as the program executes. The
+memory is updated as objects are allocated and deallocated, and the stack is
+updated as actions are performed. The state of the program can be used to track
+the progress of the program and to debug the program.
+
+#### Memory
 
 The memory is a mapping of addresses to values. The memory is used to represent
-both heap-allocated objects and also mutable parts of the procedure call stack,
-for example, for local variables. When an address is deallocated, it stays in
-memory but `!!` is printed before its value.
```

Comment:
If this part is still true, we should still mention it in the README somewhere.

Reply:
I'm not able to figure it out if this should be added or not.
For printing memory deallocation we have 
```
(+) memory-dealloc: #<allocation_index> `value`
```
Please let me know if I'm missing something.